vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -115,7 +115,7 @@
     // This enum is used to filter the displayed contacts.
     // Note: the Tchap users for who a discussion (direct chat) exists will be considered as local contacts.
     // This means they will appear in the local contacts section.
-    public enum ContactsFilter { ALL, TCHAP_ONLY, FEDERATED_TCHAP_ONLY, NO_TCHAP_ONLY }
+    public enum ContactsFilter { ALL, TCHAP_ONLY, TCHAP_ONLY_WITHOUT_EXTERNALS, TCHAP_ONLY_WITHOUT_FEDERATION, NO_TCHAP_ONLY }
```

Comment:
Maybe add some Javadoc on each enum value

Reply:
I agree this enum values will be updated in comings PRs